common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -128,6 +133,12 @@ public BrokerAcquireTokenOperationParameters brokerInteractiveParametersFromActi
                 )
         );
 
+        if (!TextUtils.isEmpty(brokerRequest.getEnvironment())) {
```

Comment:
Can this piece of code be moved somewhere else? Since the "environment' is not a field in BrokerAcquireTokenOperationParameters, could we move this piece of code somewhere else?

Reply:
Same question for similar code in method brokerSilentParametersFromBundle

Reply:
We only unpack `BrokerRequest` here , so yeah  we would have to unpack it again if it's moved and I want to avoid that. I have added a comment to be explicit

Reply:
I see, thanks for adding the comment